### ISSUE TITLE
This model is amazing!

### ISSUE THREAD

**Issue by lauridskern** [JoeyBallentine/Material-Map-Generator#6]
Hi! This model is amazing. I am wondering if you plan on further developing it. You could add more maps, eg .albedo etc. further fine tune the model etc.

 I would definitely be willing to sponsor this project!

**Comment by joeyballentine** [comment]
There was one point where I really wanted to develop this further and train a single model that could produce a ton of maps, but that ended up being really difficult and I kinda gave up. I currently spend most my time working on chaiNNer now, so I probably won't work on any successor to this for a while.

Though, since we already have normal map generation in chaiNNer, I do think it would be cool to add generators for other maps there. Maybe I'll look into that.

**Comment by lauridskern** [comment]
I see. Unfortunately, I have absolutely no idea how any of this works so I cannot contribute myself. I built something pretty cool that uses your model. Do you have Twitter or something? I would love to show it to you and maybe pay you a coffee or something

**Comment by joeyballentine** [comment]
I don't have a Twitter but I do have a ko-fi if you really wanna donate, but you really don't have to. https://ko-fi.com/jballentine

Glad you were able to use this for something useful 👍

**Comment by Trass3r** [comment]
chaiNNer looks interesting, thanks!
More generators would be cool indeed.

**Comment by ivied7** [comment]
> chaiNNer looks interesting, thanks! More generators would be cool indeed.

Yes,it's simple and useful.As same as you,looking forward to the more generators.

**Comment by pr1est0r** [comment]
Adding to the praise; You can edit line 44 in generate.py to look like this:
device = torch.device('cpu' if args.cpu else 'mps')
Makes it run on, well, MPS instead of CUDA, making full use of the GPU on a Mac.
